### DISCUSSION TITLE
Using macVlans

### DISCUSSION THREAD

**Discussion by NCVito** [haugene/docker-transmission-openvpn#2686] [Q&A]
Has anyone had any luck configuring the container to use macVlans? I loose all connectivity in and out of the container as soon as I try.

**Comment by joedyeuh** [comment]
I'm seeing the same thing, all parameters being the same macvlan fails. I tried changing the OPENVPN_OPTS as someone else suggested from restarts but then the health check fails anyways.

**Reply by joedyeuh** [reply]
solution for me was here: https://github.com/haugene/docker-transmission-openvpn/issues/1707#issuecomment-774754144

commented out "LOCAL_NETWORK"
